package.json

```diff
@@ -16,12 +16,13 @@
     "@babel/preset-env": "^7.1.5",
     "@babel/preset-react": "^7.0.0"
   },
+  "peerDependencies": {
+    "react": ">=15",
```

Comment:
It's better for react and react-dom to be peer dependencies in npm libraries so as to avoid duplicate bundling. This is how react-router does their dependency on react (see [here](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/package.json#L39))